deployments/ansible/roles/splunk-otel-collector/CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+# Contributing Guidelines
+
+If you found a bug in the Ansible role, don't hesitate to submit an issue or a 
+pull request.
+
+## Local development
+
+Testing and validation of this Ansible role is based on 
+[Ansible Molecule](https://molecule.readthedocs.io/en/latest/).
+
+### MacOS or Windows setup
+
+Development of the role can be done on MacOS or Windows machines. Make sure the
+following software is installed:
+
+- [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+- [Vagrant](https://www.vagrantup.com/downloads)
+- [Python3](https://www.python.org/downloads)
+- Python packages:
+  - ansible 
+  - molecule
+  - molecule-vagrant
+
+MacOS installation steps:
+
+```sh
+brew tap hashicorp/tap
+brew install virtualbox vagrant python3
+pip3 install ansible molecule molecule-vagrant
```

Comment:
I think having a `requirements-dev-mac.txt` and `requirements-dev-linux.txt` files would be helpful here. 
 Overall I think it's preferable to all the helpful steps as make targets instead of all the instructions living in a readme.

Reply:
Ok I've created the ticket to move it to makefile https://github.com/signalfx/splunk-otel-collector/issues/406